app/components/weekly-overview-day/template.hbs

```diff
@@ -1,5 +1,8 @@
-<div class="bar"></div>
-<div class="day">
-  {{moment-format this.day 'DD'}}
-  {{moment-format this.day 'dd'}}
+<div ...attributes title={{this.title}} role="button" {{on "click" this.click}} {{style this.style}}
+  class="weekly-overview-day {{this.classes}}">
+  <div class="bar"></div>
+  <div class="day">
+    {{moment-format @day 'DD'}}
+  {{moment-format @day 'dd'}}
+  </div>
 </div>
```

Comment:
```suggestion
<div 
  title={{this.title}}
  role="button" 
  {{on "click" this.click}} 
  {{style this.style}}
  class="weekly-overview-day 
    {{if @active "active"}}
    {{if @absence "absence"}}
    {{if @holidy "holiday"}}
    {{if (not @workday "weekend"}}"
  ...attributes
>
  <div class="bar"></div>
  <div class="day">
    {{moment-format @day 'DD'}}
    {{moment-format @day 'dd'}}
  </div>
</div>
```

Could we try it like this? Handling the classes directly in the template makes it more comprehensible. This is also [the suggested way of refactoring classNameBindings](https://github.com/emberjs/rfcs/blob/master/text/0691-deprecate-class-binding-and-class-name-bindings.md).